Code/Framework/AzCore/AzCore/Serialization/Json/JsonUtils.cpp

```diff
@@ -302,10 +302,10 @@ namespace AZ::JsonSerializationUtils
     }
 
     AZ::Outcome<void, AZStd::string> LoadObjectFromStringByType(void* objectToLoad, const Uuid& classId, AZStd::string_view stream,
-        const JsonDeserializerSettings* settings)
+        AZStd::string& deserializeErrors, const JsonDeserializerSettings* settings)
     {
         JsonDeserializerSettings loadSettings;
-        AZStd::string deserializeErrors;
+        deserializeErrors.clear();
```

Comment:
why do we need to call `.clear()` here? Might it already contain errors from a previous attempt?

Reply:
same as above. making a defensive call

Reply:
whoops, sorry for the duplicate comment. Did not mean to add more than once.